### PR TITLE
chore: add homebrew-tap recipe

### PR DESCRIPTION
**Pull request by kamilsk**
Publish homebrew-tap recipe to https://github.com/traefik/homebrew-tap

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -70,3 +70,24 @@ archives:
         format: zip
     files:
       - LICENSE
+
+brews:
+  - name: yaegi
+    caveats:
+    commit_author:
+      name:  <TODO>
+      email: <TODO>
```
Comment (kamilsk) [inline .goreleaser.yml:79]: do we need to configure this or use defaults?

File: .goreleaser.yml [hunk 2]
```diff
@@ -70,3 +70,24 @@ archives:
...
+    description: |
+      Yaegi is Another Elegant Go Interpreter.
+      It powers executable Go scripts and plugins, in embedded interpreters
+      or interactive shells, on top of the Go runtime.
+    folder: Formula
+    homepage: https://github.com/containous/yaegi
+    github:
+      owner: <TODO>
+      name:  <TODO>
```
Comment (kamilsk) [inline .goreleaser.yml:88]: and what repo I have to use to push Formula on successful builds?

**Comment by mvertes** [comment]
As we plan to migrate/rename some repos, we will apply this later, to avoid to do twice the same thing.

**Review by ldez — APPROVED** [review]
LGTM

**Pull request merged**